modules/tensor_mechanics/test/tests/ad_anisotropic_plasticity/ad_aniso_plasticity_x.i

```diff
@@ -0,0 +1,281 @@
+    family = MONOMIAL
+  []
+  [stress_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [elastic_strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
+  [disp_z]
+  []
+[]
+[AuxKernels]
+  [hydrostatic_stress]
+    type = ADRankTwoScalarAux
+    variable = hydrostatic_stress
+    rank_two_tensor = stress
+    scalar_type = Hydrostatic
+  []
+  [plasticity_strain_xx]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_xx
+    index_i = 0
+    index_j = 0
+  []
+  [plasticity_strain_xy]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_xy
+    index_i = 0
+    index_j = 1
+  []
+  [plasticity_strain_yy]
+    type = ADRankTwoAux
+    rank_two_tensor = trial_plasticity_plasticity_strain
+    variable = plasticity_strain_yy
+    index_i = 1
+    index_j = 1
+  []
+  [elastic_strain_xx]
+    type = ADRankTwoAux
+    rank_two_tensor = elastic_strain
+    variable = elastic_strain_xx
+    index_i = 1
+    index_j = 1
+  []
+  [sigma_xx]
+    type = ADRankTwoAux
+    rank_two_tensor = stress
+    variable = stress_xx
+    index_i = 1
+    index_j = 1
+  []
+[]
+
+[Functions]
+  [pull]
+    type = PiecewiseLinear
+    x = '0 1e1 1e8'
+    y = '0 -4e2 -4e2'
+  []
+[]
+
+[Modules/TensorMechanics/Master]
+  [all]
+    strain = FINITE
+    add_variables = true
+    incremental = true
+    generate_output = 'elastic_strain_xx elastic_strain_yy elastic_strain_xy stress_xx stress_xy stress_yy'
+    use_automatic_differentiation = true
+  []
+[]
+
+[Materials]
+
+  [elasticity_tensor]
+    type = ADComputeIsotropicElasticityTensor
+    youngs_modulus = 70000
+    poissons_ratio = 0.25
+  []
+
+  [elastic_strain]
+    type = ADComputeMultipleInelasticStress
+    inelastic_models = "trial_plasticity"
+    max_iterations = 50
+    absolute_tolerance = 1e-16
+  []
+
+  [trial_plasticity]
+    type = ADTransverselyIsotropicPlasticityStressUpdate
+    hardening_constant = 2000.0
+    yield_stress = 0.001 # was 200 for verification
+
+    # F G H L M N
+    hill_constants = "0.5829856 0.364424 0.6342174 2.0691375 2.3492325 1.814589"
+    absolute_tolerance = 1e-14
+    relative_tolerance = 1e-12
+    base_name = trial_plasticity
+    internal_solve_full_iteration_history = true
+    max_inelastic_increment = 2.0e-6
+    internal_solve_output_on = on_error
+  []
+[]
+
+[BCs]
+  [no_disp_x]
+    type = ADDirichletBC
+    variable = disp_x
+    boundary = left
+    value = 0.0
+  []
+
+  [no_disp_y]
+    type = ADDirichletBC
+    variable = disp_y
+    boundary = 100
+    value = 0.0
+  []
+
+  [no_disp_z]
+    type = ADDirichletBC
+    variable = disp_z
+    boundary = 101
+    value = 0.0
+  []
+
+  [Pressure]
+    [Side1]
+      boundary = right
+      function = pull
+    []
+  []
+
+[]
+
+[Preconditioning]
+  [smp]
+    type = SMP
+    full = true
+  []
+[]
+
```

Comment:
Moose has `Executioner/auto_preconditioning = true` by default now. So for Newton this block is not necessary anymore.